Website text adaptation (#145)
w3c/wot-cg: Tutorials/whatiswot/website/docs/Payloads and Headers.md

# Payloads and Headers

This is an introductory series that presents technologies and standards relevant to the Web of Things.

## Introduction

> In the Web of Things, the smart devices are constantly exchanging a lot of data. Such data is sent via payloads, therefore it is essential to understand what are payloads and headers that describe them. Thus, in this section we will cover the topics of headers and the different types of payloads - including JSON, the most used one in the scope of WoT, and its importance.

Reviewer: ```suggestion
In the Web of Things, smart devices typically exchange data with each other. Such data is sent via payloads, therefore it is essential to understand what payloads are and the headers that describe them. Thus, in this section, we will cover the topics of headers and the different types of payloads - including JSON, the most used one in the scope of WoT, and its importance.
```

> A payload should not tell what should be done by itself. We can better explain this concept with an analogy, where we consider the protocol as a truck. We can think of the payload contents as the goods that this truck is transporting. The truck's goods should not contain any information about their destination, the fact that they are transported with a truck or the color of the truck.

## What is a header?

The header is part of a message that can and should be understood by the protocol stack. All the information that the payload should not contain, goes to the header.

> Let's go back to our truck example. In that example, the header would be the label on the goods - indicating where they are being transported, what they are and etc.

![truck with parcels example](/img/tutorial/Payloads-And-Headers/truck.png)

## Header Example

This is what an HTTP header can look like. A header can contain information like request context, response context, status and it can indicate payload type.

![http header example](/img/tutorial/Payloads-And-Headers/httpheader.png)

## Payload Types

There are different payload types: JSON, XML, CBOR, text and audio to name a few.

> So let us show you an example of a **JSON payload**. 

Reviewer: In cases where we show code or json, we should not put an image but simply keep the code with correct highlighting

}
```

## Why is JSON important?

Because JSON is very relevant in the scope of the Web of Things, payloads are preferred mostly in JSON. Additionally, many other API and configuration files use JSON as well.

Here, you can see an example of a TD in JSON format. We will go into the details of how a TD should be structured and understood in a later section of this tutorial.

> 

Reviewer: For the examples below, separate them into a new section. Also adding a human-readable introduction that explains what the data is would be nice
